### ISSUE TITLE
Indenting cases inside a switch statement

### ISSUE THREAD

**Issue by echosa** [StanAngeloff/php.vim#46]
In Vim's default php syntax handling, you can set `PHP_vintage_case_default_indent` to 1 in order to have case statements indented inside switch statements. Is there an equivalent of this setting in this plugin?

**Comment by StanAngeloff** [comment]
This is interesting, I'll have to investigate as a lot of the code is inherited from the parent project. Could this be somehow related to https://github.com/2072/PHP-Indenting-for-VIm as well?

**Comment by echosa** [comment]
Looks like it is, actually. I'm fairly new to Vim so I may be confused between all these plugins. I installed Vim on my Mac through homebrew. One of the files installed with Vim itself is `/usr/local/share/vim/vim74/indent/php.vim`, which is apparently the file from that Github repo. I can run `:help php-indent` and I see the help for that. However, when I set that `PHP_vintage_case_default_indent` setting, select a region with visual mode, then indent with =, the case statements still do not indent inside their switch statement.

Am I doing something wrong? Is there a way to see what = is doing so I can see if it's running the right function or something? I come from Emacs, and that's normally the first thing I would do if something wasn't working right. Perhaps this question is better asked in that other repo. I'll take my question there if you don't want to answer it (or don't know the answer).
